README.md

```diff
@@ -208,6 +208,33 @@ oktaOidc.signOutOfOkta(authStateManager, from: self) { error in
 ```
 Sample app [example](https://github.com/okta/samples-ios/blob/master/browser-sign-in/OktaBrowserSignIn/SignInViewController.swift#L62-L74)
 
+### signOut
+
+This method helps to perform mutli-step sign out flow. Provide options that you want to perform and sdk will run them as a batch. Available options are:
+- revokeAccessToken - sdk revokes access token
+- revokeRefreshToken - sdk revokes refresh token
+- signOutFromOkta - sdk calls [`signOutOfOkta`](#signoutofokta)
+- removeTokensFromStorage - sdk removes tokens from the secure storage
+- revokeTokensOptions - revokes access and refresh tokens
+- allOptions - revokes tokens, signs out from Okta and removes tokens from the secure storage
+
+The order of operations performed by the sdk:
+1. Revoke access token, if option is set
+2. Revoke refresh token, if option is set
+3. Browser sign out, if option is set
+4. Remove tokens from the secure storage, if option is set
+
+```swift
+let options: OktaSignOutOptions = .revokeTokensOptions
+options.insert(.signOutFromOkta)
+oktaOidc.signOut(with: options, authStateManager: authStateManager, from: self, callback: { success, notFinishedOptions, error in
+    if !success {
+        // Error
+        return
+    }
+})
+```
+
 ### authenticate
 
 If you already logged in to Okta and have a valid session token, you can complete authorization by calling `authenticate(withSessionToken:)`. In case of successful authorization, this operation will return valid `OktaOidcStateManager` in its callback. Clients are responsible for further storage and maintenance of the manager.
```

Comment:
If you already signed in to Okta and have a valid session token, you can complete authorization by calling `authenticate(withSessionToken:)`. Upon successful authorization, this operation returns a valid `OktaOidcStateManager` in the callback. Clients are responsible for further storage and maintenance of the manager.